delphi/translators/for2py/pyTranslate.py

```diff
@@ -1142,6 +1143,92 @@ def get_python_source(self):
 
         return "".join(self.pyStrings)
 
+    """
+        This function checks the type of a variable and returns the appropriate python syntax type name
+    """
+    def get_type(self, node):
+        variable_type = node["type"].lower()
+        if variable_type in TYPE_MAP: 
+            return TYPE_MAP[variable_type]
+        else:
```

Comment:
Please make this an if-elif-else construct (instead of an if-else with another if-else nested inside it).